Documentation/tutorial/docfx.exe_user_manual.md

```diff
@@ -296,6 +297,74 @@ _disableAffix         | bool    | Indicate whether to show the affix bar on the
 _disableContribution  | bool    | Indicate whether to show the `View Source` and `Improve this Doc` buttons.
 _gitContribute        | object  | Customize the `Improve this Doc` URL button for public contributors. Use `repo` to specify the contribution repository URL. Use `branch` to specify the contribution branch. Use `path` to specify the folder for new overwrite files .
 
+#### 3.2.3 Separated metadata files for global metadata and file metadata
+
+Take global metadata for example, you can set `globalMetadataFiles` in `docfx.json` or `--globalMetadataFiles` in build command line. The usage of `fileMetadataFiles` is the same as `globalMetadataFiles`.
+
+There're some metadata file examples:
+
++ globalMetadata file example 
+    ```json
+    {
+        "_appTitle": "DocFX website",
+        "_enableSearch": "true"
+    }
+    ```
+
++ fileMetadata file example
+    ```json
+    {
+        "priority": {
+            "**.md": 2.5,
+            "spec/**.md": 3,
+        },
+        "keywords": {
+            "obj/docfx/**": ["API", "Reference"],
+            "spec/**.md": ["Spec", "Conceptual"],
+        }
+    }
+    ```
+
+There're some examples about how to use separated metadata files.
+
++ use `globalMetadataFiles` in `docfx.json`
+    ```json
+    ...
+    "globalMetadataFiles": ["global1.json", "global2.json"],
+    ...
+    ```
+
++ use `--globalMetadataFiles` in build command line
+    ```
+    docfx build --globalMetadataFiles global1.json,global2.json
+    ```
+
++ use `fileMetadataFiles` in `docfx.json`
+    ```json
+    ...
+    "fileMetadataFiles": ["file1.json", "file2.json"],
+    ...
+    ```
+
++ use `--fileMetadataFiles` in build command line
+    ```
+    docfx build --fileMetadataFiles file1.json,file2.json
+    ```
+
+One more thing need note is that, the metadata set in command line will merge with the metadata set in `docfx.json`. 
```

Comment:
>One more thing need note is that [](start = 0, length = 32)

One more thing need note is that -> "One more thing need to note is that" or "Note that" directly?